docs/features.md

```diff
@@ -152,85 +152,51 @@ donejs deploy
+#### How it works
 
-Building for mobile is simple with the steal-cordova plugin. First, install the plugin:
-````
-npm install steal-cordova --save-dev
-````
-And build your application:
-```
-var stealTools = require("steal-tools");
+For iOS and Android builds, DoneJS integrates with [Apache Cordova](https://cordova.apache.org/) to generate an "app" version that is ready to be uploaded to the App Store/Google Play.
 
-var cordovaOptions = {
-  buildDir: "./build/cordova",
-  id: "com.hello.world",
-  name: "HelloWorld",
-  platforms: ["ios", "android"],
-  index: __dirname + "/index.html"
-};
-
-var stealCordova = require("steal-cordova")(cordovaOptions);
-var buildPromise = stealTools.build({
-  config: __dirname + "/package.json!npm"
-});
+For native desktop applications, DoneJS integrates with [NW.js](https://github.com/nwjs/nw.js) to create an native OSX, Windows, or Linux application.
 
-buildPromise.then(stealCordova.build);
-```
-DoneJS can also build native desktop applications using [NW.js](https://github.com/nwjs/nw.js) and steal-nw. NW.js allows you to run Node.js modules directly from the DOM, unleashing the potential of your application. Building to desktop is as simple as building to mobile. Install:
-```
-npm install steal-nw --save-dev
-```
-and build:
-```
-var stealTools = require("steal-tools");
-var stealNw = require("steal-nw");
+Adding this integration is as simple as running `donejs add cordova` or `donejs add nw` and running your build script.
 
-var nwOptions = {
-  buildDir: "./build",
-  platforms: ["osx"],
-  files: ["./**/*"]
-};
+[Follow the guide](./Guide.html#section_Desktopandmobileapps) to see an example in action.
 
-var buildPromise = stealTools({
-  config: __dirname + "/package.json!npm"
-});
+### Supports All Browsers, Even IE8
 
-buildPromise.then(function(buildResult){
-    stealNw(nwOptions, buildResult);
-});
-```
-It's that easy! Building to iOS, Android and desktop expands your potential audience and can make your app more powerful. DoneJS makes this quick and easy, with minimal configuration, all using your web application's codebase.
+DoneJS applications support Internet Explorer 8 without any additional effort. You can even write applications using [most ES6 features](http://babeljs.io/docs/advanced/caveats/) that run on IE8, using the built-in babel integration.
 
-### Supports All Browsers, Even IE8
+Many people don't care about this because IE8 is on its way out, which is a very good thing! 
 
-DoneJS is compatible with modern and legacy versions of all browsers, including Chrome, Firefox, Safari, and Internet Explorer back to version 8. While other frameworks eschew legacy browser support, DoneJS embraces it. Doing so maximizing compatibility, and thus potential audience adoption, of your app.
+But its not quite dead yet, at still over 3% of the browser market in September 2015. For many mainstream websites, banks, and ecommerce applications, IE8 continues to hang around the browser stats. 
```

Comment:
@julia-allyce I got rid of that chart because it wasn't very impressive, I'll explain when we talk